diff --git a/code/backend/Cleanuparr.Infrastructure/Features/Jobs/DownloadCleaner.cs b/code/backend/Cleanuparr.Infrastructure/Features/Jobs/DownloadCleaner.cs
index 83550a63..286f2f5e 100644
--- a/code/backend/Cleanuparr.Infrastructure/Features/Jobs/DownloadCleaner.cs
+++ b/code/backend/Cleanuparr.Infrastructure/Features/Jobs/DownloadCleaner.cs
@@ -127,7 +127,7 @@ public sealed class DownloadCleaner : GenericHandler
         if (downloadServiceToDownloadsMap.Count > 0)
         {
             // wait for the downloads to appear in the arr queue
-            await Task.Delay(TimeSpan.FromSeconds(10), _timeProvider, cancellationToken);
+            await Task.Delay(TimeSpan.FromMilliseconds(500), _timeProvider, cancellationToken);
 
             await ProcessArrConfigAsync(ContextProvider.Get<ArrConfig>(nameof(InstanceType.Sonarr)), true);
             await ProcessArrConfigAsync(ContextProvider.Get<ArrConfig>(nameof(InstanceType.Radarr)), true);
